### PR TITLE
Fix warnings/lower deployment target

### PR DESCRIPTION
**Pull request by gfontenot**

### CODE REVIEW

**Comment by steam** [comment]
:+1:

File: LlamaKit/Result.swift [hunk 1]
```diff
@@ -54,12 +54,12 @@ public func failure<T,E>(error: E) -> Result<T,E> {
 /// Construct a `Result` using a block which receives an error parameter.
 /// Expected to return non-nil for success.
 
-public func try<T>(f: NSErrorPointer -> T?, file: String = __FILE__, line: Int = __LINE__) -> Result<T,NSError> {
+public func try<T>(file: String = __FILE__, line: Int = __LINE__, f: NSErrorPointer -> T?) -> Result<T,NSError> {
```
Comment (tonyd256) [inline LlamaKit/Result.swift]: weird .. why did this move?

Comment (steam) [inline LlamaKit/Result.swift]: New swift warning `Closure parameter prior to parameters with default arguments will not be treated as a trailing closure`

Comment (tonyd256) [inline LlamaKit/Result.swift]: ha .. that is annoying

**Pull request merged**